README.md

```diff
@@ -24,6 +20,10 @@
     </a>
 </p>
 
+<p align="center">
+  Node.js SDK for <a href="https://github.com/ossf/scorecard">OpenSSF scorecard</a>
+</p>
```

Comment:
Imo redondant as this is already in the banner (even if the link to the ossf/scorecard repo is missing in the banner)

Reply:
I left it in because it redirects to ossfs GitHub, but I'm removing it immediately